singer/metadata.py

```diff
@@ -24,7 +24,7 @@ def get(compiled_metadata, breadcrumb, k):
 
 def get_standard_metadata(schema=None, schema_name=None, key_properties=None,
                           valid_replication_keys=None, replication_method=None):
-    mdata = {}
+    mdata = {(): {}}
```

Comment:
I think this change is not needed. The way `metadata.write()` works, it will make this `()` key and its value will be `{}`

Reply:
And what if none of the `if` conditions are met?

Reply:
Ha, you're right. So I changed the test 😂 

In all seriousness though, I want to avoid changing the function because it's in use in the wild already and changing the shape of the "empty response" is a breaking change in my opinion

Reply:
Yeah, that's a totally fair point. It's similar to the point/question I made above (https://github.com/singer-io/singer-python/pull/132#issuecomment-718697548) though I came to the same conclusion as you on that one!